docs/install.rst

```diff
@@ -184,6 +184,30 @@ globally.
    available globally. This helps avoid issues caused by hiding the public
    schema from queries.
 
+Working with Tenant specific schemas
+====================================
+Since each Tenant has it's own database you need a way to tell Django what
```

Comment:
Each tenant doesn't have it's own database – that language (at least for me) implies an entry in `settings.DATABASES` for each tenant.

We should say that each tenant has a schema in the database.